addons/survey/static/src/views/fields/boolean_toggle/survey_boolean_toggle_field_no_saving.js

```diff
@@ -0,0 +1,16 @@
+/** @odoo-module **/
+import { registry } from "@web/core/registry"
+import { BooleanToggleField } from '@web/views/fields/boolean_toggle/boolean_toggle_field';
+
+export class SurveyBooleanToggleFieldNoSaving extends BooleanToggleField {
```

Comment:
Have to say it is a bit verbose ^^

Reply:
@ampa-odoo , in that case I would advise to drop the `Boolean` which doesn't add much .

Reply:
@flch-odoo Done with the changes. Please check it out!!